### PR TITLE
Add Pull Request Template

### LINKED ISSUE #301: Create PR template for contributions
Might be nice to have a PR template (something with description of problem, changes made... screenshots etc) so that it will be easier for others to context switch and review changes.

### PR DESCRIPTION
**Pull request by tkwidmer**
# Context
- Fixes #301

# Summary of Changes

- Adds pull request template file

# Checklist

- [n/a] Tested Mobile Responsiveness
- [n/a] Added Unit Tests
- [n/a] CI Passes
- [n/a] Deploys to Heroku on test Correctly (Maintainers will handle)

# Screenshots

## Before
n/a
## After
n/a

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- change_1_here
+- change_2_here
+
+# Checklist
+
+- [ ] Tested Mobile Responsiveness
+- [ ] Added Unit Tests
+- [ ] CI Passes
+- [ ] Deploys to Heroku on test Correctly (Maintainers will handle)
```
Comment (mi-wood) [inline .github/PULL_REQUEST_TEMPLATE.md]: checkbox for documentation. both code and service documentation when required

Comment (dannypaz) [inline .github/PULL_REQUEST_TEMPLATE.md]: +1

**Comment by dannypaz** [comment]
@tkwidmer love this (and will definitely help new engineers to be more transparent)

**Comment by vernistage** [comment]
This is great. Thanks!

**Comment by tkwidmer** [comment]
I'm going to merge this

**Pull request merged**